CMakeLists.txt

```diff
@@ -676,8 +693,7 @@ target_include_directories(${APP_PROJECT_NAME} SYSTEM
 )
 
 # Use the Widgets module from Qt 5.
-qt5_use_modules(${APP_PROJECT_NAME} Widgets)
-qt5_use_modules(${APP_PROJECT_NAME} Network)
+target_link_libraries(${APP_PROJECT_NAME} Qt5::Network Qt5::Widgets)
```

Comment:
This should also work properly when the environment variable `Qt5_DIR` is correctly set (or passed to cmake). Where does your `Qt5_DIR` point to?

Reply:
qt5_use_modules was flagged as deprecated, that’s why I changed this.

Reply:
Ah ok, thanks for clarification!